packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -255,45 +259,41 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                         headers: Object.keys(headers).length !== 0 ? true : undefined,
                         waitQueueLength: this.epochTracker.rateLimiter.waitQueueLength,
                     },
-                    async () => this.epochTracker.fetchAndParseAsJSON<IBlob>(url, { headers }, FetchType.blob),
+                    async () => this.epochTracker.fetchResponse(url, { headers }, FetchType.blob),
                 );
             });
-            blob = response.content;
+            buffer = await response.arrayBuffer();
```

Comment:
@ChumpChief - I'm a bit confused here, maybe I missed something in your PR. I see we call  this.scheduleClearBlobsCache() at the top, but we never cache this blob. Given that we do not cache, should we not reset timer?